programs/programs/ma/pe/member.py

```diff
@@ -154,3 +154,33 @@ class MaHeadStart(PolicyEngineMembersCalculator):
         *dependency.irs_gross_income,
     ]
     pe_outputs = [dependency.member.HeadStart]
+
+
+class MaCsfp(CommoditySupplementalFoodProgram):
+    """
+    Massachusetts Commodity Supplemental Food Program (CSFP) calculator.
+    Extends the federal CSFP calculator with MA state code dependency and county filtering.
+    Only available in specific MA counties served by Greater Boston Food Bank.
+    """
+
+    eligible_counties = [
```

Comment:
Do we have a source for this county list? Might be nice to include it in a comment here.